src/app/Dashboard/components/DashboardContainer.js

```diff
@@ -80,8 +99,17 @@ export class DashboardContainer extends Component {
           </div>
           <div className='user-dashboard__actions col-sm-12'>
             <h3 className='user-dashboard__title'>My Activities</h3>
+            <LogACtivityForm
+              className='modal'
```

Comment:
What's this prop for?

Reply:
@Kachulio1 The props found in this component are used to open and close the LogPointsModal

Reply:
I'm asking this className='modal'

Reply:
I'm asking this className='modal'

Reply:
@Kachulio1 That className was doing nothing and unnecessary doing nothing